lib/Zonemaster/CLI.pm

```diff
@@ -1,15 +1,14 @@
 # Brief help module to define the exception we use for early exits.
 package Zonemaster::Engine::Exception::NormalExit;
-use 5.014002;
+use v5.26;
```

Comment:
Does that mean that we increase the lowest Perl version of Zonemaster from v5.16 to v5.26?